src/index.js

```diff
@@ -6,31 +6,43 @@ import Redbox from 'redbox-react';
 import Routes from './routes';
 import './index.css';
 
-ReactDOM.render(
-  <AppContainer errorReporter={Redbox}>
-    <Routes />
-  </AppContainer>,
-  document.getElementById('root')
-);
+if (process.env.NODE_ENV !== 'development') {
```

Comment:
Do we want to make this stronger and actually `production|staging`? Just thinking about test, etc.